source/community.applications/usr/local/emhttp/plugins/community.applications/javascript/helpers.js

```
@@ -151,78 +131,70 @@ function post(options,callback) {
   }
-  postCount++;
-  console.log("Post Count: "+postCount);
-  if ( typeof callback === "function" ) {
-    $.post(execURL,options,function(result){
-       try {
+  
+    
+  $.post(execURL,options).done(function(result) {
+    if (result.script) {
+      try {
+        eval(result.script);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)